eng/scripts/update-version.ps1

```diff
@@ -0,0 +1,144 @@
+<#
+.SYNOPSIS
+    Updates the version for a specific Azure SDK package.
+.DESCRIPTION
+    This script updates the package version by invoking the update-version 
+    command from @azure-tools/js-sdk-release-tools.
+    
+    The script validates the package path and runs the version update tool.
+.PARAMETER SdkRepoPath
+    The absolute path to the root folder of the local SDK repository.
+.PARAMETER PackagePath
+    The absolute path to the root folder of the local SDK project (package).
+    Must contain a valid package.json file.
+.PARAMETER ReleaseType
+    The type of release: 'beta' or 'stable'.
+.PARAMETER Version
+    The version number to set (e.g., '1.0.0' or '1.0.0-beta.1').
+.PARAMETER ReleaseDate
+    The release date in YYYY-MM-DD format.
+.EXAMPLE
+    .\update-version.ps1 `
+        -SdkRepoPath "D:\GithubSource\tmpSource\azure-sdk-for-js" `
+        -PackagePath "D:\GithubSource\tmpSource\azure-sdk-for-js\sdk\storage\arm-storage" `
+        -ReleaseType "stable" `
+        -Version "1.0.0" `
+        -ReleaseDate "2025-11-13"
+    
+    Updates the version for the arm-storage package.
+.NOTES
+    - Requires @azure-tools/js-sdk-release-tools to be available via npx.
+    - The tool will update package.json and CHANGELOG.md with the new version.
+#>
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true, HelpMessage = "Absolute path to the SDK repository root")]
+  [string]$SdkRepoPath,
+  
+  [Parameter(Mandatory = $true, HelpMessage = "Absolute path to the SDK package directory")]
+  [string]$PackagePath,
+  
+  [Parameter(Mandatory = $true, HelpMessage = "Release type: 'beta' or 'stable'")]
```

Comment:
Both `ReleaseType` and `Version` parameters are not required, but one of them must be provided.